.github/workflows/integration.yaml

```
@@ -206,8 +206,8 @@ jobs:
       - name: Run chart-testing (install)
         if: steps.list-changed.outputs.changed == 'true'
         run: |
-          docker build -t docker.io/kubeflow/spark-operator:local .
-          minikube image load docker.io/kubeflow/spark-operator:local
+          docker build -t ghcr.io/kubeflow/spark-operator:local .
+          minikube image load ghcr.io/kubeflow/spark-operator:local
```

Please can you keep this image for the Spark Operator image:
```suggestion
          docker build -t ghcr.io/kubeflow/spark-operator/spark-controller-manager:local .
          minikube image load ghcr.io/kubeflow/spark-operator/spark-controller-manager:local
```

That will allow users quickly recognize what project that image belongs to:
```
ghcr.io/kubeflow/<PROJECT_NAME>/<COMPONENT_NAME>
```
Ref discussion: https://cloud-native.slack.com/archives/C073N7B6K3R/p1742475222915929?thread_ts=1741893292.124179&cid=C073N7B6K3R

Does this image sound good to you @ChenYi015 @vara-bonthu @jacobsalway ?

---

I prefer a shorter image name if possible. Since the Spark operator currently has only one component image, there is no need to add a suffix to distinguish it. We can make that change if more components are added in the future.

---

@ChenYi015 What do you think about this ?
```
ghcr.io/kubeflow/spark-operator/controller-manager
```

---

@andreyvelich I agree with @ChenYi015 Lets keep the same name. We can change it later once we decide to go with more components.

---

@vikas-saxena02 @ChenYi015 Why I suggested that we do it now, because we've done the same of other Kubeflow projects, like
- Model Registry: https://github.com/kubeflow/model-registry/pull/899
- Katib: https://github.com/kubeflow/katib/pull/2529

I believe, it would be easier for users to understand relationship between images and Kubeflow projects if we keep this consistent.

---

@andreyvelich I understand with what you are suggesting but given the new docker.io rate limits will kick in from 1st April i.e. another 3 days. Can we go ahead with change of image registry as there is no input from reviewers about it? I am more than happy to make another PR for the change of repo name suggested by you and discuss this in the next spark-operator community call? @ChenYi015 can you please share your thoughts as we are approaching a deadline here. cc @vara-bonthu @jacobsalway

---

I am ok to move this forward, but using the consistent image name is super easy change that we can make in this PR, rather than changing it in the future and cleanup the legacy images.

I don't see any issue to keep this image for the Spark Operator:
```
ghcr.io/kubeflow/spark-operator/controller-manager
```

But I leave the final lgtm to the Spark Operator maintainers: @vara-bonthu @ChenYi015 @kubeflow/wg-training-leads @jacobsalway @yuchaoran2011 @ImpSy @nabuskey

---

@juliusvonkohout this is the discussion which we are having about changing repo name along with image registry url. If other people agree to it, I am more than happy to make it.

---

I like having the consistent image naming scheme for the organization. If we are making changes to registry locations, we may as well do it now than to do it later. Changing registry, then changing naming scheme later will cause annoyance / bad experience for the end users imo.

`controller-manager` is a bit confusing though. Maybe keep it simple with just `controller`?

---

Thanks @nabuskey . @andreyvelich  are you happy with just `controller` as suggested? If yes, I can quickly  update the same!!

---

@ChenYi015 any inputs here will really be appreciated

---

Sure, controller also sounds good:
```
ghcr.io/kubeflow/spark-operator/controller
```

---

> Sure, controller also sounds good:
> 
> ```
> 
> ghcr.io/kubeflow/spark-operator/controller
> 
> ```

Also sounds good to me.

---

@juliusvonkohout @andreyvelich @nabuskey @ChenYi015 the change has been made